LICENCE.txt

```diff
@@ -0,0 +1,190 @@
+EUROPEAN UNION PUBLIC LICENCE v. 1.2
```

Comment:
I suggest looking at OPENEUROPA-547 regarding this topic.

Reply:
Thanks @aritomelo we will deal with it on all projects once OPENEUROPA-547 will be in the next sprint.

Reply:
Okidocki!